src/components/Header.tsx

```diff
@@ -8,17 +8,50 @@ export const Header = () => {
   const navigate = useNavigate();
   return (
     <header className="App-header">
-      <div className="header">
+      <div className="HeaderMainPage">
+
         <Button onClick={() => navigate('/railway/')}>
           <img className="App-logo" src={Logo} alt="Logo" />
         </Button>
+        
+        <nav className='Menu-bar'>
+          <ul>
+          <li><a href="#" className='click'>Ticket Purchase</a>
+            <div className='Sub-menu-1'>
+            <ul className='menu-list'>
```

Comment:
reuse menu component from mui library, examples of use are here 
https://mui.com/material-ui/react-menu/#basic-menu